### ISSUE TITLE
Accessibility issue with pager

### ISSUE THREAD

**Issue by vever001** [openeuropa/oe_bootstrap_theme#277]
The pager currently has accessibility issues, here is the report from AXE (used during behat on SRB):

```
│ -
│   description: 'Ensures links have discernible text'
│   help: 'Links must have discernible text'
│   helpUrl: 'https://dequeuniversity.com/rules/axe/4.4/link-name?application=axeAPI'
│   id: link-name
│   impact: serious
│   nodes:
│     -
│       all: {  }
│       any:
│         -
│           data: null
│           id: has-visible-text
│           impact: minor
│           message: 'Element does not have text that is visible to screen readers'
│           relatedNodes: {  }
│         -
│           data: null
│           id: aria-label
│           impact: serious
│           message: 'aria-label attribute does not exist or is empty'
│           relatedNodes: {  }
│         -
│           data: null
│           id: aria-labelledby
│           impact: serious
│           message: 'aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty'
│           relatedNodes: {  }
│         -
│           data:
│             messageKey: noAttr
│           id: non-empty-title
│           impact: serious
│           message: 'Element has no title attribute'
│           relatedNodes: {  }
│       failureSummary: |-
│         Fix all of the following:
│           Element is in tab order and does not have accessible text
│ 
│         Fix any of the following:
│           Element does not have text that is visible to screen readers
│           aria-label attribute does not exist or is empty
│           aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
│           Element has no title attribute
│       html: '<a class="page-link" href="?page=4"><svg class="bi icon--fluid"><use xlink:href="/themes/contrib/oe_bootstrap_theme/assets/icons/bcl-default-icons.svg#chevron-double-right"></use></svg></a>'
│       impact: serious
│       none:
│         -
│           data: null
│           id: focusable-no-name
│           impact: serious
│           message: 'Element is in tab order and does not have accessible text'
│           relatedNodes: {  }
│       target:
│         - '.page-item:nth-child(7) > .page-link[href="?page=4"]'
│   tags:
│     - cat.name-role-value
│     - wcag2a
│     - wcag412
│     - wcag244
│     - section508
│     - section508.22.a
│     - ACT
│ 
Critical/serious HTML Accessibility error(s) found. (Exception)
```

BCL allows passing aria labels since v0.18.0, see [here](https://github.com/openeuropa/bootstrap-component-library/commit/fb27d027cc298afbd9d4410eca876169da1ca0ae).

Perhaps we can simply pass hardcoded aria labels in `templates/overrides/navigation/pager.html.twig`?

**Comment by brummbar** [comment]
Solved in #304 .
